### DISCUSSION TITLE
Results Offline?

### DISCUSSION THREAD

**Discussion by nbrandont** [birchill/10ten-ja-reader#1459] [Q&A] [answered]
This may be a dumb question, but was curious. Are the translation requests and results always calculated offline? Was wondering if I could load up an html doc from my desktop and still be able to translate the text in it without a connection.

**Comment by birtles** [comment, marked as answer]
Hi! It's not a dumb question at all! Yes, the results are always calculated offline so you don't need an Internet connection. An Internet connection is only needed when you first install the add-on to download the latest dictionary data. It then tries to download any updated definitions twice a week but it will keep working even without these updates.

**Reply by nbrandont** [reply]
Awesome, thanks for the reply and for your work on the extension! It's been very helpful.
